clients/render/index.js

```diff
@@ -38,23 +38,27 @@ domReady(function () {
 		//Build configurations
 		document.querySelectorAll('.cf2-field-wrapper').forEach(function (fieldElement) {
 			const fieldIdAttr = fieldElement.getAttribute('data-field-id');
+			if( 'undefined' !== typeof idAttr ){
```

Comment:
Inconsistent spacing between `if` and `(` and the next char. It's different here and everywhere else in the file.

Reply:
Ok, I removed all spaces in condition statements